ci/Vagrantfile

```diff
@@ -0,0 +1,40 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+  config.ssh.username = "root"
+  config.vm.box = "fedora/RELEASE-cloud-base"
+  config.vm.synced_folder "./", "/vagrant", type: "virtualbox"
+
+  config.vm.define "controller" , primary: true do |controller|
+    controller.vm.provider :virtualbox do |vb|
```

Comment:
We're using VirtualBox as the VM provider? Not KVM? ... Oh, we're running on MacOS.

Reply:
Github actions enabled Nested Virtualization only in MacOS.

See: https://github.com/actions/virtual-environments/issues/433 and https://github.com/actions/virtual-environments/issues/433#issuecomment-603934964